pkg/core/blockchain.go

```diff
@@ -815,6 +856,15 @@ func (bc *Blockchain) storeBlock(block *block.Block, txpool *mempool.Pool) error
 		mpt.Collapse(10)
 	}
 
+	err, ok := <-aerdone
+	if ok && err != nil {
```

Comment:
I think for `!ok` we always have `err == nil`, so we may replace it with a single `err := <-aerdone`.

Reply:
Fixed.